### PR TITLE
79 create endpoints for activity

### PR DESCRIPTION
**Pull request by DeepakR-Sastry**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const dotenv = require("dotenv");
 
 ///////////////////////////////////////////// Configurations ////////////////////////////////////////
 const User = require('./models/user.model');  // register schema for model User
+const Activity = require('./models/activity.model')
```
Comment (Sooraj-beep) [inline server/app.js]: what is the purpose of having this constant? it does not seem to be used anywhere in this file. If no purpose, please get rid of it.

File: server/models/activity.model.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+////////////// Import dependencies //////
+const mongoose = require('mongoose');
+// const bcrypt = require('bcrypt');
+const Schema = mongoose.Schema;
+////////////////////////////////////////
+
+
+//////// Create a database model //////
+const ActivitySchema = new Schema({
```
Comment (Sooraj-beep) [inline server/models/activity.model.js:9]: doesnt the Activity also contain a post object? which is a list of post ids?

File: server/routes/activity.route.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+router.get('/',validateJwtToken, activityController.getActivity);
+
+// router.route('/')
+// .put(activityController.updateActivity);
+// router.route('/')
+// .delete(activityController.deleteActivity);
+// router.route('/')
+// .get(activityController.getActivity)
+// router.route('/getActivities')
+// .get(activityController.getActivity)
```
Comment (Sooraj-beep) [inline server/routes/activity.route.js]: get rid of commented code int this file

File: server/routes/activity.route.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+router.stack.forEach(l => console.log(l.route.path, l.route.methods))
```
Comment (Sooraj-beep) [inline server/routes/activity.route.js]: get rid of this console log

**Review by Sooraj-beep — COMMENTED** [review]
Also include testcases for all endpoints ( 1 passing and 1 failing for each operation would be ideal with extra boundary condition testcases if you can think of any)

File: server/models/activity.model.js (hunk 2 above)
Comment (DeepakR-Sastry) [reply server/models/activity.model.js:9]: I have already intimated that the post objects aren't added because we don't have a model for it yet; in the standup last week I told that I could go back and add post objects once we create a model for it

Comment (Sooraj-beep) [reply server/models/activity.model.js:9]: you could still include in the schema but send a null value for now

**Review by Sooraj-beep — COMMENTED** [review]
Can you also include test cases in this commit? (1 passing and 1 failing for each operation would be ideal with extra boundary condition test cases if you can think of any)
